tools/generate-component/boilerplate/component.stories.ts

```diff
@@ -36,6 +44,8 @@ const meta: Meta = {
     actions: {
       handles: [__nameUpperCase__.events.myEventName],
     },
+    backgroundColor: (context: StoryContext) =>
+      context.args.negative ? '#484040' : 'var(--sbb-color-white)',
```

Comment:
I think it's better to have black here in the example. Might still be changed in the implementation
```suggestion
      context.args.negative ? 'var(--sbb-color-black)' : 'var(--sbb-color-white)',
```